### PR TITLE
fix(modal/toast/banner/card): align slot names

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**
Aligned bottom slot name and renamed actions prop (modal).

**Solving issue**  
Fixes: [CDEP-2447](https://tegel.atlassian.net/browse/CDEP-2447)

**How to test**  
1. Go to the affected components
2. Make sure the slots work as inteded.

[CDEP-2441]: https://tegel.atlassian.net/browse/CDEP-2441?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CDEP-2447]: https://tegel.atlassian.net/browse/CDEP-2447?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: core/src/components/banner/banner.tsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ export class TdsBanner {
   render() {
     const usesHeaderSlot = hasSlot('subheader', this.host);
     const usesSubheaderSlot = hasSlot('subheader', this.host);
-    const usesBottomSlot = hasSlot('bottom', this.host);
```
Comment (nathalielindqvist) [inline core/src/components/banner/banner.tsx:102]: No link visible in the Banner, looks like it still uses "bottom" slot in the stories file?
image

File: core/src/components/card/card.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { generateUniqueId, hasSlot } from '../../utils/utils';
  * @slot thumbnail - Slot for the Card thumbnail.
  * @slot body - Slot for the body section of the Card.
  * @slot body-image - Slot for the body section of the Card, used for image.
- * @slot bottom - Slot for the bottom section of the Card.
+ * @slot actions - Slot for the bottom section of the Card, used for buttons .
```
Comment (nathalielindqvist) [inline core/src/components/card/card.tsx:10]: Looks like the "bottom" slot is still used in the stories file for the Card component?
image

File: core/src/components/toast/toast.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { generateUniqueId, hasSlot } from '../../utils/utils';
 /**
  * @slot header - Slot for the Toast header.
  * @slot subheader - Slot for the Toast subheader.
- * @slot bottom - Slot for the Toast bottom section, used for links.
+ * @slot actions - Slot for the Toast bottom section, used for links.
```
Comment (nathalielindqvist) [inline core/src/components/toast/toast.tsx:8]: No link content in Toast component, looks like it still uses the "bottom" slot in the stories file?
image

File: core/src/components/card/card.tsx [hunk 4]
```diff
@@ -7,7 +7,7 @@ import { generateUniqueId, hasSlot } from '../../utils/utils';
  * @slot thumbnail - Slot for the Card thumbnail.
  * @slot body - Slot for the body section of the Card.
  * @slot body-image - Slot for the body section of the Card, used for image.
- * @slot bottom - Slot for the bottom section of the Card.
```
Comment (nathalielindqvist) [inline core/src/components/card/card.tsx:10]: Do we want to update the slot name for the icon in Storybook? 
image

Comment (nathalielindqvist) [inline core/src/components/card/card.tsx:10]: Link part disappeared completely now, is there maybe some place that still calls for the "bottom" slot in Card component?
image

**Review by nathalielindqvist — APPROVED** [review]
Great work! 🔥

**Review by alexandereneroth — APPROVED** [review]

**Pull request merged**